src/CLI/IAction.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CLI
+{
+    interface IAction
+    {
+        public Task Execute();
```

Comment:
For interfaces, visibility modifiers are not needed;
```suggestion
        Task Execute();
```